coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -92,6 +92,11 @@
         'args': '--new-window',
         'gui': True
     },
+    'vscode': {
+        'file_arg_template': '{filename}:{line}',
```

Comment:
Is `line` working now? Because last time you mentioned that it was misbehaving for `code`.

Reply:
No, still the code works with 'code' but not with 'vscode'.

Reply:
But line is working with 'code'
should I change 'vscode' back to 'code'

Reply:
In this case, you can make an alias by using `alias vscode="code"` in your terminal and then check if the changes work or not.

Reply:
Just rename to `code` Code is used everywhere for VSCode, even its twitter handle

Reply:
I tried that but it didn't work @sangamcse